docpipeline/generate.py

```diff
@@ -22,6 +22,7 @@
 from google.cloud import storage
 from google.oauth2 import service_account
 from google.protobuf import text_format, json_format
+from docpipeline.prepare_java import prepare_java_toc
```

Comment:
What do you think of naming the module `prepare` instead of `prepare_java`? Then, we can also move `add_prettyprint` into that file.

Reply:
Agreed, generate.py should just be about generating.